### ISSUE TITLE
“SoapUI 5.6.0 Installer” can’t be opened because Apple cannot check it for malicious software

### ISSUE THREAD

**Issue by Nathan-Smith** [SmartBear/soapui#591]
Downloading the latest version of SoapUI from the Website (https://www.soapui.org/downloads/soapui/)
Attempts to open the Installer results in:
![image](https://user-images.githubusercontent.com/1488676/113238559-bfa6fe80-92f4-11eb-88b4-8b635845330e.png)
Trying to bypass this within System Preferences -> Security & Privacy yields no difference
![image](https://user-images.githubusercontent.com/1488676/113238575-c7ff3980-92f4-11eb-8e15-762e0531bef5.png)

Running macOS Big Sur 11.2.3

**Comment by kevdhan** [comment]
I too have this same issue!

**Comment by Arax250190** [comment]
I'm too

**Comment by jstolp** [comment]
Me too... :(

**Comment by NoS1ze** [comment]
Same issue here

**Comment by DavidIjsud** [comment]
same issue here

**Comment by sagerman4** [comment]
Was able to get around this issue by installing with homebrew using the `--no-quarantine` flag.

```
> brew install soapui --no-quarantine
```

**Comment by Nathan-Smith** [comment]
@sagerman4 thanks that worked!

```
➔ brew install soapui --no-quarantine
==> Downloading https://s3.amazonaws.com/downloads.eviware/soapuios/5.6.0/SoapUI-5.6.0.dmg
######################################################################## 100.0%
==> Installing Cask soapui
Warning: macOS's Gatekeeper has been disabled for this Cask
==> Running installer script 'SoapUI 5.6.0 Installer.app/Contents/MacOS/JavaApplicationStub'
The installation directory has been set to /Applications.
Extracting files ...
Finishing installation ...
🍺  soapui was successfully installed!
```

**Comment by arekdist** [comment]
This should be no solution. It's only workaround. It is unacceptable SoapUI app is not signed at all:

```
codesign -dv /Applications/SoapUI.app
/Applications/SoapUI.app: code object is not signed at all

spctl -a -vvv /Applications/SoapUI.app
/Applications/SoapUI.app: rejected
source=no usable signature
```

Not to mention modern MacOS (Catalina, Big Sur) requires a valid code signing, but also notarization!

**Comment by doilio** [comment]
Also having this issue. 
Using Big Sur 11.5.2

**Comment by michaeltookes** [comment]
This worked for me as well using @sagerman4 suggestion. Thanks!

**Comment by ReadyAPIAgent** [comment]
5.7.0 will address this issue. coming soon. Sorry for inconveniences.

**Comment by jsoref** [comment]
Fwiw, I believe this is macOS saying that the app needs a notarization: https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution

(A feature required by Catalina as of June 2019.)

**Comment by ReadyAPIAgent** [comment]
you're right, we have already notarized current SNAPSHOT build and it works fine. The internal issue is closed, broader audience will get it next year, mid January.

**Comment by ReadyAPIAgent** [comment]
fixed in 5.7.0

**Comment by qaiscvs** [comment]
I found the fix for it. we need to replace javaFX version 16 jar files with version 20.0.2.https://repo1.maven.org/maven2/org/openjfx/
